patches/minecraft/net/minecraft/entity/EntityCreature.java.patch

```diff
@@ -0,0 +1,28 @@
+--- ../src-base/minecraft/net/minecraft/entity/EntityCreature.java
++++ ../src-work/minecraft/net/minecraft/entity/EntityCreature.java
+@@ -15,6 +15,7 @@
+     private BlockPos field_70775_bC = BlockPos.field_177992_a;
+     private float field_70772_bD = -1.0F;
+     private final float field_184661_bw = PathNodeType.WATER.func_186289_a();
++    protected float maxLeashDistance = 10.0F;
```

Comment:
This should have a comment on the same line like `// forge: make maxLeashDistance a field instead of hard-coded` so that it gets migrated correctly to new Minecraft versions. Just in case vanilla ends up changing this code at all, the intention is more clear with a comment.

Reply:
That actually makes a lot of sense and directly answers some of my questions about how forge deals with patches when mc updates. I've fixed it.